### DISCUSSION TITLE
Error message: 'Failed to install from GitHub'

### DISCUSSION THREAD

**Discussion by Emmanuel-Faure** [RangeShifter/RangeshiftR-tutorials#2] [Q&A]
Hi everyone, 

I'm trying to install RangeShiftR on my computer (Windows 64-bits, R version 4.0.3) but I keep receiving this error message after launching the command `devtools::install_github("https://github.com/RangeShifter/RangeShiftR-package", ref = "main")` : 

`Failed to install 'RangeShiftR' from GitHub:
  (converted from warning) installation of package ‘C:/Users/faureemm/AppData/Local/Temp/RtmpcDrV6k/file4743aae4037/RangeShiftR_1.0.0.tar.gz’ had non-zero exit status`

Prior to this I've successfully installed Rcpp, devtools and Rdpack packages.

Do you know what might be the problem, and how to fix it?

**Comment by orb16** [comment]
This fails because a warning is converted to an error. You would need to check the warning, but, one solution might be to use `Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS="true")` and then use the remotes package to install (see here for the package - it's similar to devtools: https://github.com/r-lib/remotes).

**Reply by Emmanuel-Faure** [reply]
Thanks a lot! 
Finally the problem seemed to come from my internet firewall, which was blocking some features of the R package. So I just had to download the package from GitHub on my computer, and it went perfectly fine!
